app/src/main/java/com/dvm/appd/oasis/dbg/MainActivity.kt

```diff
@@ -180,13 +180,22 @@ class MainActivity : AppCompatActivity(), NetworkChangeNotifier {
                         .setPositiveButton(
                             resources.getString(R.string.alert_notification_positive_button),
                             DialogInterface.OnClickListener { dialog, which ->
-                                sharedPreferences.edit().putBoolean("wantsNotification", false).apply()
+                                sharedPreferences.edit().putBoolean("wantsNotification", false)
+                                    .apply()
                                 try {
                                     startActivity(intent)
-                                }catch (e: ActivityNotFoundException){
-                                    Toast.makeText(applicationContext, "Error. Not supported in your phone", Toast.LENGTH_SHORT).show()
+                                } catch (e: ActivityNotFoundException) {
+                                    Toast.makeText(
+                                        applicationContext,
+                                        "Error. Not supported in your phone",
+                                        Toast.LENGTH_SHORT
+                                    ).show()
```

Comment:
This is the correct indentaion for a Toast message, but because of the theme already followed throughout the app, make it in a single line. Try to keep your indentation uniform with the rest of the app